caluma-demo/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:8
+
+COPY package.json yarn.lock /app/
+
+WORKDIR /app
+
+RUN yarn
+
+COPY . /app
+
+RUN yarn ember deploy production
+
+FROM nginx:alpine as final
+
+COPY --from=0 /app/tmp/deploy-dist /usr/share/nginx/html
```

Comment:
Why do we name the last stage (but don't seem to use it) and don't name the first stage which we use here? Would be nice to do it consistently.

Reply:
Ah right. I wanted to rename the stages and forgot it. Now they are renamed.